python/triqs_cthyb/tail_fit.py

```diff
@@ -24,6 +24,103 @@
 
 from triqs.gf.gf_fnt import fit_hermitian_tail_on_window, replace_by_tail
 
+from triqs.operators import c, c_dag
+from triqs.atom_diag import trace_rho_op
+
+def comm(A,B): return A*B - B*A
+def anticomm(A,B): return A*B + B*A
+
+def sigma_high_frequency_moments(density_matrix,
+                           ad_imp, 
+                           gf_struct, 
+                           h_int):
+    """
+    Calculate the first and second high frequency moment of Sigma_iw
+    following Rev. Mod. Phys. 83, 349 (2011). They read
+    (0) Sigma_0       =   0
+    (1) Sigma_1       = -<{[Hint,c],c+}> (Hartree-Fock)
```

Comment:
@harrisonlabollita The 0th moment of the self-energy should contain the Hartree shift

Reply:
Ahh yes, I think I misunderstood. The TRIQS notation is the following:

G(iwn) = c0/(iwn)^0 + c1/(iwn)^1 + c2/(iwn)^2 + ....
c0 = 0, c1 = 1, etc.

Sigma(iwn) = c0/(iwn)^0 + c1/(iwn)^1 + c2/(iwn)^2 + ....
c0 = Sigma_HartreeFock, etc.